static/app/components/pagination/paginationCaption.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+type Props = {
+  caption: React.ReactNode;
+};
+
+function PaginationCaption({caption}: Props) {
+  return <Wrapper>{caption}</Wrapper>;
+}
+
+export default PaginationCaption;
+
+const Wrapper = styled('span')`
+  color: ${p => p.theme.gray400};
```

Comment:
Should this be using a color alias so you get the correct contrast in dark mode as well?